network-api/networkapi/mozfest/templates/partials/hero/large.html

```diff
@@ -0,0 +1,14 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<picture>
+  {% image page.banner fill-4032x2304 as image_xl %}
```

Comment:
this partial only differs in the "vertical" resolution, but Django templates don't do preprocessing macros or something similar to `calc()`, so there's no way to only make those vertical dimension numbers something that gets controlled by a single value that is passed into the partial.

At least, not without writing more code that adds additional django logic in a tag file which then needs to load another template file, so it just gets too impractical and instead I opted to duplicate the hero image code with large and regular banners, to allow "homepage" vs. "every other page" styling.